index.d.ts

```diff
@@ -0,0 +1,6 @@
+export class Instance extends WebAssembly.Instance {
+  constructor(module: WebAssembly.Module, imports: WebAssembly.Imports);
```

Comment:
I think you don't need to override the constructor either. In the future we might want to make upstream definitions more precise, and then we'd have to override here as well, but for now it's unnecessary.

Reply:
Ah, good catch!